ui/app/templates/components/pki-cert-show.hbs

```diff
@@ -45,7 +45,7 @@
           @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}}
           @value={{date-format (get this.model attr.name) "MMM dd, yyyy hh:mm:ss a" isFormatted=true}}
         />
-      {{else if (and (get this.model attr.name) (eq attr.name "revocationTime"))}}
```

Comment:
`(get this.model attr.name)` returns `0` so this block wasn't being hit and `Revocation time: 0` was displaying in the UI. Now zero is passed to the `date-format` helper which errors so revocation time isn't displayed unless it has a date value